odoo/addons/base/models/ir_cron.py

```diff
@@ -102,7 +102,9 @@ def _process_jobs(cls, db_name):
                     if not job:
                         break
                     _logger.debug("job %s acquired", job['id'])
-                    cls._process_job(db, cron_cr, job)
+                    # enable overridings of _process_job() on that database
+                    Cron = odoo.registry(db_name)[cls._name]
+                    Cron._process_job(db, cron_cr, job)
```

Comment:
May I suggest to make that part of the code like it was in Odoo 14.0 (https://github.com/odoo/odoo/blob/14.0/odoo/addons/base/models/ir_cron.py#L237-L238)?
I find it slightly more understandable.
```suggestion
                    registry = odoo.registry(db_name)
                    registry[cls._name]._process_job(db, cron_cr, job)
```